adhocracy4/follows/static/follows/__tests__/FollowButton.jest.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react'
+import { render, fireEvent, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
+import { FollowButton } from '../FollowButton'
+import jQuery from 'jquery'
+import api from '../../../../static/api'
+window.$ = window.jQuery = jQuery
```

Comment:
why do we need jquery in react test?

Reply:
ah, is left over from mocking the ajax call in the api, will remove